docker-compose.yml

```diff
@@ -293,8 +293,6 @@ services:
       /entrypoint.sh ls localhost:50151 opi_api.network.evpn_gw.v1alpha1.BridgePortService.UpdateBridgePort -l && \
       echo toremove && \
       apt update && apt install iproute2 -y && \
-      ip link add br-tenant type bridge vlan_default_pvid 0 vlan_filtering 1 vlan_protocol 802.1Q && \
```

Comment:
Is this being removed since the configuration will be done through the API?  Will this have implications on the CI testing and automation that need to be handled?

Reply:
The only reason that we remove that is because the br-tenant is created now when the opi-evpn-br starts. That means that there is no need to create the br-tenant beforehand as docker compose command. It is not created through API but is created with linux netlink command when the Linux General Module (lgm) starts. There will be no implecations on the CI testing and automation.

https://github.com/opiproject/opi-evpn-bridge/blob/d5501d68225a2f43412850c3eff355a78f449670/pkg/LinuxGeneralModule/lgm.go#L331